src/Console_TEST.cc

```diff
@@ -89,7 +90,7 @@ TEST_F(Console_TEST, NoInitAndLog)
   std::string path;
   EXPECT_TRUE(common::env(GZ_HOMEDIR, path));
   path = common::joinPaths(path, logPath);
-  EXPECT_TRUE(common::removeAll(path));
+  common::removeAll(path);
```

Comment:
I'm curious why this `EXPECT_TRUE` was removed.

Reply:
Added a note in [ab269e8](https://github.com/gazebosim/gz-common/pull/615/commits/ab269e8eee1c0bc837f3940f6dda25f66842ba69).